### PR TITLE
[MM-12591] Add scroll animation for click on more unreads in LHS

### PR DESCRIPTION
**Pull request by sudheerDev**
#### Summary
Add a scroll animation to jump on click of more unreads

#### Ticket Link
[MM-12951](https://mattermost.atlassian.net/browse/MM-12951)

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] Ran `make check-style` to check for style errors (required for all pull requests)
- [x] Ran `make test` to ensure unit and component tests passed
- [x] Has UI changes

nov-16-2018 03-37-40
nov-16-2018 03-37-32

### CODE REVIEW

**Comment by sudheerDev** [comment]
Hey, @wiersgallak I added a new library with BSD licence similar to other facebook licence we have

Can you help me out on the process here?

From what i understand from previous context is to add the contents of https://github.com/facebook/rebound-js/blob/master/LICENSE to NOTICE.txt with the existing format similar to other libraries in the file.

Is that it?

**Comment by wiersgallak** [comment]
@sudheerDev  Yes we will need to add a record in the NOTICE.txt file in the format similar to existing.  Please note this as a BSD-3-Clause license.  Let me know if you have questions or want me to review it once you record it.

File: package.json [hunk 1]
```diff
@@ -45,6 +45,7 @@
     "react-redux": "5.0.7",
     "react-router-dom": "4.3.1",
     "react-select": "2.0.0",
+    "rebound": "^0.1.0",
```
Comment (saturninoabril) [inline package.json]: Please have it on fix version.

**Comment by sudheerDev** [comment]
@wiersgallak Changes made. 
>Please note this as a BSD-3-Clause license

The library is under BSD licence  so kept the same

**Pull request merged**